File: Fibonacci/fib.pl
fib(0, 1):-!.
fib(1, 1):-!.
fib(N, Fib):-
	N1 is N-1,
	N2 is N-2,
	fib(N1, FN1),
	fib(N2, FN2),
	Fib is FN1+FN2.

File: Fibonacci/fib.c
#include<stdio.h>

void main(){
	int n=40;
	printf("fib(%d) = %d\n", n, fib(n));
}

int fib(int n){
	if(n==0 || n==1){
		return 1;
	}
	return fib(n-1)+fib(n-2);
}
